utils/api.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+export const fetchArticles = (article_id) => {
+  let endpointString = "https://nc-news-uld9.onrender.com/api/articles";
```

Comment:
you can use axios.create here instead - take a look at the notes

Reply:
OK - added axios.create and refactored rest of the code